planning/planning_index.adoc

```diff
@@ -107,6 +108,8 @@ amount of memory used for systemd itself.
 NOTE: As of docker-1.10, the docker run parameter of _--privileged_ is no longer needed to use systemd
 within a container.
 
+You can implement using [systemd fairly simply in your dockerfile](creating_using_systemd).
```

Comment:
This is markdown syntax. You want `link:#creating_using_systemd[systemd fairly simply in your dockerfile].`